store/postgres/src/relational.rs

```diff
@@ -460,6 +472,35 @@ impl Layout {
         Ok(())
     }
 
+    pub fn update_metadata(
```

Comment:
Should this be `update_unversioned` to keep the naming consistent?

Reply:
Good point. Renaming it.